code/modules/clothing/suits/marine_armor.dm

```diff
@@ -289,6 +289,15 @@
 	supporting_limbs = CHEST | GROIN | ARM_LEFT | ARM_RIGHT | HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT | FOOT_LEFT | FOOT_RIGHT //B18 effectively stabilizes these.
 	resistance_flags = UNACIDABLE
 
+/obj/item/clothing/suit/storage/marine/specialist/equipped(mob/user, slot)
+	. = ..()
+	if (GLOB.monitor_statistics.B18_in_use.Find(src))
+		GLOB.monitor_statistics.B18_in_use+=src
+
+/obj/item/clothing/suit/storage/marine/specialist/unequipped(mob/user, slot)
+	. = ..()
+	GLOB.monitor_statistics.B18_in_use-=src
```

Comment:
why not increment and deincrement?

Reply:
Because equipped is called when you put an item in your hand, not when you pick it up.
While unequipped is called only when it's dropped on the ground. 

Just incrementing will makes the counter going up each time you shuffle your b18. Same with the guns

Reply:
use |= here then